concert_halls/data/website_view.xml

```diff
@@ -0,0 +1,168 @@
+        <field name="name">Contact Us</field>
+        <field name="key">concert_halls.contactus</field>
+        <field name="type">qweb</field>
+        <field name="active" eval="True"/>
+        <field name="website_id" ref="website.default_website" />
+        <field name="arch" type="xml">
+            <t name="Contact Us" t-name="website.contactus">
+                <t t-call="website.layout">
+                    <t t-set="logged_partner" t-value="request.env['website.visitor']._get_visitor_from_request().partner_id" />
+                    <t
+                        t-set="contactus_form_values"
+                        t-value="{                 'email_to': res_company.email,                 'name': request.params.get('name', ''),                 'phone': request.params.get('phone', ''),                 'email_from': request.params.get('email_from', ''),                 'company': request.params.get('company', ''),                 'subject': request.params.get('subject', ''),             }"
+                    />
+                    <span class="hidden" data-for="contactus_form" t-att-data-values="contactus_form_values" />
+                    <div id="wrap" class="oe_structure oe_empty">
+                        <section class="s_title parallax s_parallax_is_fixed bg-black-50 pt24 pb24 o_colored_level" data-vcss="001" data-snippet="s_title" data-scroll-background-ratio="1">
+                            <span class="s_parallax_bg oe_img_bg" style="background-image: url('/web/image/concert_halls.configurator_1_s_sidegrid_default_image_3'); background-position: 50% 0;" />
+                            <div class="o_we_bg_filter bg-black-50">
+                                <br />
+                            </div>
+                            <div class="container">
+                                <h1>Contact us</h1>
+                            </div>
+                        </section>
+                        <section class="s_text_block pt40 pb40 o_colored_level" data-snippet="s_text_block">
+                            <div class="s_allow_columns container">
+                                <div class="row">
+                                    <div class="col-lg-7 mt-4 mt-lg-0 o_colored_level">
+                                        <h3>Opening Hours</h3>
+                                        <p>Monday to Friday — 10:00 a.m. to 6:00 p.m.</p>
+                                        <p>
+                                            📅 <em>Summer Closure:</em> Closed from July 1 to September 1, 2025<br />
+                                            <br />
+                                        </p>
+                                        <h3>Get in Touch</h3>
+                                        <p>
+                                            📞 <strong>Phone:</strong> +32 81 81 37 00<br />
+                                            📧 <strong>Email:</strong> <a href="mailto:info@odoo.be">****@example.com</a>
```

Comment:
Update the info@odoo.be">****@example.com also please confirm for static values with PO -
 **<em>Summer Closure:</em> Closed from July 1 to September 1, 2025<br />**